### DISCUSSION TITLE
GoPro 9 black please help me

### DISCUSSION THREAD

**Discussion by Wumpscut13** [gopro/labs#794] [General]
Hello there. Could someone help me write code to make a GoPro 9 start recording when motion is detected and stop recording after 4 seconds of no motion? I'm aiming for 2.7K resolution at 60 frames per second. I feel like I'm doing something wrong as the camera seems to turn off after a few minutes and doesn't respond to motion in the frame. I want to record birds or animals in the park. What am I doing wrong? QR Command: mV0r27p60!SM5-5D4H4!R

**Comment by dnewman-gpsw** [comment]
The command looks fine.  Experiment at home first to determine the sensitivity.  Animals will need to be fairly near.   Camera turning off, could your settings, you can disable auto off in the menu or scan ```oC0```.

**Reply by Wumpscut13** [reply]
"you can disable auto off in the menu or scan oC0." 

Thank you for the response. Can I do this in the QRControl app?

**Reply by dnewman-gpsw** [reply]
Yes, under "Prefs", Auto Off, select Never

**Reply by Wumpscut13** [reply]
Thank you :)

**Comment by Wumpscut13** [comment]
I have one more question. Does motion detection on the GoPro 9 require the LCD to be on at all times?

**Reply by dnewman-gpsw** [reply]
No, that was only a HERO10 bug.

**Comment by Wumpscut13** [comment]
Does the motion detection function also work when the camera is, for example, attached to a tripod or lying still on a table? Sorry for my lame questions but I still have a problem. My GoPro only detects motion for a while, when the screen turns off it stops detecting motion. What am I doing wrong?

**Reply by dnewman-gpsw** [reply]
It only works when stationary, so tripod or table is ideal.  No idea what the issue you are seeing is, but some quick testing should be easy enough. I tested a HERO9 and it worked fine, LCD has timed out, but it still started recording when I place fingers in front of the lens. Is the camera turning off?  Did you disable the camera's auto off?
